.github/workflows/add_staff_label.yml

```
@@ -0,0 +1,23 @@
+name: Move new issues to inbox
```

Maybe something like:

```suggestion
name: Add Staff Label
```

---

Thank you! Copy paste error 😅